crates/sequencer/src/mempool.rs

```diff
@@ -81,16 +81,14 @@ impl<C: sov_modules_api::Context> CitreaMempool<C> {
             ..pool_config
         };
 
-        Ok(Self(Pool::eth_pool(
-            TransactionValidationTaskExecutor::eth(
-                client,
-                Arc::new(chain_spec),
-                blob_store,
-                TokioTaskExecutor::default(),
-            ),
-            blob_store,
-            pool_config,
-        )))
+        let validator = TransactionValidationTaskExecutor::eth_builder(Arc::new(chain_spec))
+            .no_cancun()
```

Comment:
I am curious was to how this affected us previously

Reply:
I guess the mempool would expect blob transactions but since we passed noop blob store never return them.